docs-src/concepts/what-is-the-temporal-platform.md

```diff
@@ -9,12 +9,92 @@ tags:
-Together these components create a runtime for Workflow Executions.
+Together these components create a runtime for Workflow Executions; that is-they create a runtime for your application.
 
 ![The Temporal Platform](/diagrams/temporal-platform-simple.svg)
 
-The Temporal Cluster is open source and can be operated by you.
-The Temporal Cloud is a set of Clusters operated by us.
+The Temporal Cluster is open source and can be operated by you. [Temporal Cloud](/concepts/what-is-temporal-cloud) is a set of Clusters operated by us.
 
-Worker Processes are hosted by you and execute your code.
+Worker Processes are hosted and operated by you and execute your code.
 They communicate with a Temporal Cluster via gRPC.
+
+![Basic component topology of the Temporal Platform](/diagrams/temporal-platform-component-topology.svg)
+
+### Temporal versus traditional
+
+In a traditional system, the service exists to spawn function executions.
+The Temporal Platform exists to facilitate [Workflow Executions](/workflows#workflow-execution).
+
+![Temporal vs Traditional system](/diagrams/temporal-vs-traditional.svg)
+
+Although the two systems seem similar at first glance, they differ in several significant ways.
+
+**Failure**
+
+With a traditional system, a service function execution is both volatile and short-lived.
+
+- If a function execution fails, it's not resumable because all execution state is lost.
+  The longer a function execution awaits, the higher the chance of failure.
+- A traditional function execution typically has a limited lifespan, often measured in minutes.
+
+With Temporal, a Workflow Execution is resumable.
+
+- A Workflow Execution is fully resumable after a failure.
+- Temporal imposes no deadlines on Workflow Executions.
+
+**State**
+
+With a traditional system, stoppage or failure means that all execution state is lost.
+Your application (or a supporting component) must monitor the service's response to initiate a retry of the service execution.
+A retry starts from its _initial_ state.
+
+With Temporal, computation resumes from its _latest_ state.
+All progress is retained.
+
+**Communication**
+
+With a traditional system, you can't communicate with a function execution.
+
+With Temporal, [Signals](/concepts/what-is-a-signal) and [Queries](/concepts/what-is-a-query) enable data to be sent to or extracted from a Workflow Execution.
+
+**Scope**
+
+With a traditional system, a service function execution can at best represent a business process.
+Typically, it represents only a part of a business process.
+
+A Temporal Workflow Execution can represent a business process or an entire business object.
+
+### Failure mitigation
+
+The Temporal Platform addresses both platform-level failures and application-level failures.
+
+#### Platform-level failure
+
+A platform-level failure refers to a failure that occurs within the underlying platform or infrastructure that supports an application or system.
```

Comment:
```suggestion
A platform-level failure is a failure that occurs within the underlying platform or infrastructure that supports an application or system.
```
If you want to emphasize the term itself, an alternative is "_Platform-level failure_ refers to a failure...."